lib/ckb/api.rb

```diff
@@ -40,29 +46,22 @@ def initialize(host: CKB::RPC::DEFAULT_URL, mode: MODE::TESTNET)
         dao_cell_data = CKB::Utils.hex_to_bin(system_cell_transaction.outputs_data[2])
         dao_code_hash = CKB::Blake2b.hexdigest(dao_cell_data)
 
-        set_dao_cell(dao_out_point, dao_code_hash)
+        set_dao_dep(dao_out_point, dao_code_hash)
       end
     end
 
     # @param out_point [CKB::Types::OutPoint]
-    # @param code_hash [String] "0x..."
-    def set_system_script_cell(out_point, code_hash)
-      @system_script_out_point = out_point
-      @system_script_code_hash = code_hash
+    # @param secp_cell_type_hash [String] 0x...
+    def set_secp_group_dep(out_point, secp_cell_type_hash)
+      @secp_group_out_point = out_point
+      @secp_cell_type_hash = secp_cell_type_hash
```

Comment:
Why not set `secp_cell_code_hash` here.

Reply:
because I don't want put `secp_cell_code_hash` to `set_secp_group_dep`

Reply:
ok